papaya-core/src/main/java/pl/edu/agh/papaya/service/sprint/SprintCreationWizard.java

```diff
@@ -49,7 +49,7 @@ public SprintCreationWizard withDurationPeriod(LocalDateTimePeriod durationPerio
     public Sprint create() {
         Sprint sprint = new Sprint();
 
-        if (!enrollmentPeriod.isBefore(durationPeriod)) {
+        if (enrollmentPeriod.isAfter(durationPeriod)) {
             throw new IllegalStateException("enrollmentPeriod must be strictly before durationPeriod");
```

Comment:
to zmieniłbym też treść wyjątku

Reply:
Idealne issue zreviewowałeś, bo akurat tu mam commity z całej trójcy chwilowo (żeby wygodniej się deployowało). Amendowanie po trzech branchach ma szansę mnie zabić, ale będę walczyć :stuck_out_tongue: